config/zones/QA.yaml

```diff
@@ -4,47 +4,48 @@ bounding_box:
   - - 52.11654707100007
     - 26.660101630000042
 capacity:
+  _comment: Emission factor calculations are available at https://docs.google.com/spreadsheets/d/1ownS3qse6zDG4gNnKEZDroIKLNiqZcULqp34LJUxrL0/edit?usp=sharing
```

Comment:
This is not allowed in our current pydantic config, but I think we should change that rather than removing it.

Reply:
Since we don't show this anywhere, should we just make it a code comment like this instead?
```suggestion
# Capacity calculations are available at https://docs.google.com/spreadsheets/d/1ownS3qse6zDG4gNnKEZDroIKLNiqZcULqp34LJUxrL0/edit?usp=sharing
```

Reply:
> Since we don't show this anywhere, should we just make it a code comment like this instead?

Yes but there is a risk some other tool or something will remove it if it's run as a part of a script. Up to you IMO but would be really annoying if that happened and we would probably end up needing to fix it anyway.

Reply:
I moved the comment to top-level, as I am unsure if having the key lower might break some existing functionality that is only expecting actual production modes.